src/mod2c_core/units.c

```diff
@@ -539,15 +510,6 @@ void unit_init() {
 diag("Bad MODLUNIT environment variable. Cant open:", s);
 		}
 	}
-	if (!inpfile && (inpfile = fopen(dfile, "r")) == (FILE *)0) {
```

Comment:
In coreneuron we always define MODUNIT env variable (instead of using auto detection logic based on neuronhome()).